inc/classes/subscriber/third-party/plugins/class-smush-subscriber.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace WP_Rocket\Subscriber\Third_Party\Plugins;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Subscriber for compatibility with Smush
+ *
+ * @since  3.4.2
+ * @author Soponar Cristina
+ */
+class Smush_Subscriber implements Subscriber_Interface {
+	/**
+	 * Subscribed events for Smush.
+	 *
+	 * @since  3.4.2
+	 * @author Soponar Cristina
+	 * @inheritDoc
+	 */
+	public static function get_subscribed_events() {
+		if ( ! defined( 'WP_SMUSH_VERSION' ) ) {
+			return [];
+		}
+
+		return [
+			'update_option_wp-smush-settings'      => [ 'smush_maybe_deactivate_lazyload', 11 ],
+			'activate_wp-smushit/wp-smush.php'     => [ 'smush_maybe_deactivate_lazyload', 11 ],
+			'rocket_maybe_disable_lazyload_helper' => 'is_smush_lazyload_active',
+		];
+	}
+
+	/**
+	 * Disable WP Rocket lazyload when activating WP Smush and values are already in the database.
+	 *
+	 * @since  3.4.2
+	 * @author Soponar Cristina
+	 */
+	public function smush_maybe_deactivate_lazyload() {
```

Comment:
```suggestion
	public function maybe_deactivate_rocket_lazyload() {
```

I think it's better to convey the goal of the method